eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -164,7 +164,8 @@ jobs:
 
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
         parameters:
-          Directory: sdk/${{ parameters.ServiceDirectory }}
+          Directory: ''
+          Urls: (git diff origin/master HEAD --name-only -- *.md)
```

Comment:
Is this the correct diff? What if this pipeline runs against a feature branch? This will produce a potentially large diff with a lot of files. I think we are going to need to figure out the correct set files to diff for pull requests as well as the correct set of changes if running outside of a pull request (i.e. manual, CI, scheduled)